app/gui2/src/components/CircularMenu.vue

```diff
@@ -1,24 +1,40 @@
 <script setup lang="ts">
+import SvgIcon from '@/components/SvgIcon.vue'
 import ToggleIcon from '@/components/ToggleIcon.vue'
 
 const props = defineProps<{
   isOutputContextEnabledGlobally: boolean
   isOutputContextOverridden: boolean
   isDocsVisible: boolean
   isVisualizationVisible: boolean
+  isFullMenuVisible: boolean
 }>()
 const emit = defineEmits<{
   'update:isOutputContextOverridden': [isOutputContextOverridden: boolean]
   'update:isDocsVisible': [isDocsVisible: boolean]
   'update:isVisualizationVisible': [isVisualizationVisible: boolean]
+  startEditing: []
+  openFullMenu: []
```

Comment:
I think the "full menu visible" could be v-model also, similarly to isVisualizaitonVisible, isDocsVisible and all others.

So, not `openFullMenu` but `update:isFullMenuVisible`

Reply:
This was actually no longer used, so is just removed now.